src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -169,13 +170,21 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
             block_sorting_parameters=scheme2_sorting_parameters, block_duration_sec=p["scheme3_block_duration_sec"]
         )
 
-        scheme = p["scheme"]
-        if scheme == "1":
-            sorting = ms5.sorting_scheme1(recording=recording, sorting_parameters=scheme1_sorting_parameters)
-        elif p["scheme"] == "2":
-            sorting = ms5.sorting_scheme2(recording=recording, sorting_parameters=scheme2_sorting_parameters)
-        elif p["scheme"] == "3":
-            sorting = ms5.sorting_scheme3(recording=recording, sorting_parameters=scheme3_sorting_parameters)
+        assert isinstance(recording, BaseRecording)
```

Comment:
What is this assert doing? is there a concern that users wouldn't supply a recording. Should we tell the user what they put in wrong?

Reply:
This is to keep the linter happy. The output of `load_recording_from_folder` is `BaseExtractor | None`, whereas mountainsort5 expects a BaseRecording. I will update this to make it more clear.